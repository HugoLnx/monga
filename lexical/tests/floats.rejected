927.356e-
927.356e+
927.356e.
927.356e-f
.e
.f
.e3
+.
.+
