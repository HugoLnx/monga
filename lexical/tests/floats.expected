FLOAT,0.45315
FLOAT,398000.00000
TK_MINUS,
FLOAT,9.40000
FLOAT,927356.00000
FLOAT,927356.00000
TK_MINUS,
FLOAT,0.92730
TK_PLUS,
FLOAT,1200.00000
FLOAT,0.90000
FLOAT,9.00000
FLOAT,90.00000
FLOAT,900.00000
FLOAT,1.00000
TK_SEMICOLON,
FLOAT,927.35600
TK_ID,rty
FLOAT,900.00000
TK_ID,xpto
FLOAT,927356.00000
TK_PLUS,
FLOAT,927356.00000
TK_PLUS,
FLOAT,927.35600
TK_ID,ef
FLOAT,927.35600
TK_ID,ef3
FLOAT,90.00000
